src/main/scala/net/snowflake/spark/snowflake/pushdowns/querygeneration/AggregationExpression.scala

```diff
@@ -0,0 +1,41 @@
+package net.snowflake.spark.snowflake.pushdowns.querygeneration
+
+import org.apache.spark.sql.catalyst.expressions._
+import org.apache.spark.sql.catalyst.expressions.aggregate._
+
+/**
+  * Extractor for aggregate-style expressions.
```

Comment:
nit: you're inconsistent with your comments, sometimes you start in the first line, sometimes in the second